src/components/Comment/Comment.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from "react";
 import { css, cx } from "emotion";
 
-import { neutral, space, palette } from "@guardian/src-foundations";
+import { space, palette } from "@guardian/src-foundations";
+import { neutral } from "@guardian/src-foundations/";
```

Comment:
You should import from `@guardian/src-foundations/palette` - the fact you can import from the root is a bug

Reply:
good catch